qutebrowser/utils/urlutils.py

```diff
@@ -298,14 +298,28 @@ def qurl_from_user_input(urlstr):
     else:
         ipstr = urlstr.strip()
         rest = ''
+
+    # This may error out because we first use this function to validate the
+    # options (_validate_all) before the register the config...
+    try:
+        default_scheme = config.get('network', 'scheme-order')[0]
+    except KeyError:
+        default_scheme = None
```

Comment:
It seems to me that in that case, you would get an error from another piece of code somewhere anyway?